app/narwos.py

```diff
@@ -755,6 +758,9 @@ def get_exec_filtered_report():
 def get_detailed_filtered_report():
 
     report_type = request.form.get('detailed_filtered_report_type')
+    if not report_type:
```

Comment:
ooo thats right, thats the case. You see, you do not need to execute code further because you have catched no report type at the beginning. You can also easily return specified message